src/main/java/gregtech/common/metatileentities/MetaTileEntities.java

```diff
@@ -530,6 +532,7 @@ public static void init() {
 
         STEAM_OVEN = registerMetaTileEntity(1023, new MetaTileEntitySteamOven(gregtechId("steam_oven")));
         STEAM_GRINDER = registerMetaTileEntity(1024, new MetaTileEntitySteamGrinder(gregtechId("steam_grinder")));
+        STEAM_BLAST_FURNACE = registerMetaTileEntity(1025, new MetaTileEntitySteamBlastFurnace(gregtechId("steam_blast_furnace")));
```

Comment:
Conflicting ids

Reply:
Fixed, but adds a small ID Shift that only affects large miners